### ISSUE TITLE
Learn more about accessibility links - Accessibility Principles page

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-quick-start#20]
I think we want to add a link to <http://www.w3.org/WAI/intro/people-use-web/principles>. I think this is an *awesome* introduction to accessibility requirements. I think this should be the second link. I know it's a sub-page of How PWDs Use the Web; however, I think people might not get to it. Maybe we just link to the Principles page, and hope they find the others if they want?

**Comment by iadawn** [comment]
I agree that this is a great resource. I am hesitant to not link to the other aspects of the How PWDs Use the Web resource as I think it is extremely important and beneficial for folks new to accessibility.

**Comment by slhenry** [comment]
OK, link to both :-)

**Comment by iadawn** [comment]
I have added these in, feel free to check the summary wording I have used
